README.md

```diff
@@ -62,3 +65,12 @@ be necessary to do this. Debuggers other than gdb may work but are untested.
 
 Processes running in the emulated kernel can be introspected through a simple
 debug console available via telnet on port 12347.
+
+## Profiling
+
+Mikage uses [Tracy](https://github.com/wolfpld/tracy) for profiling. To enable it,
+make sure to tell Conan to enable profiling in the conanfile.py and rebuild Mikage.
+Get your Tracy server (you can usally install it via your package manager) up and
```

Comment:
Tracy doesn't actually ship in that many distributions. Fedora and openSUSE don't have it.

```suggestion
Get your Tracy server (often available from the system package manager) up and
```